app/components/markdown/markdown_latex_block/index.tsx

```diff
@@ -0,0 +1,232 @@
+            padding: 6,
+            position: 'absolute',
+            right: 0,
+            top: 0,
+        },
+        languageText: {
+            color: theme.sidebarHeaderTextColor,
+            fontSize: 12,
+        },
+        errorText: {
+            fontSize: 14,
+            marginHorizontal: 5,
+            color: theme.errorTextColor,
+        },
+    };
+});
+
+const LatexCodeBlock = ({content, theme}: Props) => {
+    const intl = useIntl();
+    const insets = useSafeAreaInsets();
+    const managedConfig = useManagedConfig<ManagedConfig>();
+    const styles = getStyleSheet(theme);
+
+    const handlePress = preventDoubleTap(() => {
+        const screen = Screens.LATEX;
+        const passProps = {
+            content,
+        };
+
+        const languageDisplayName = getHighlightLanguageName('latex');
+        const title = intl.formatMessage({
+            id: 'mobile.routes.code',
+            defaultMessage: '{language} Code',
+        }, {
+            language: languageDisplayName,
+        });
+
+        Keyboard.dismiss();
+        requestAnimationFrame(() => {
+            goToScreen(screen, title, passProps);
+        });
+    });
+
+    const handleLongPress = useCallback(() => {
+        if (managedConfig?.copyAndPasteProtection !== 'true') {
+            const renderContent = () => {
+                return (
+                    <View
+                        testID='at_mention.bottom_sheet'
+                        style={styles.bottomSheet}
+                    >
+                        <SlideUpPanelItem
+                            icon='content-copy'
+                            onPress={() => {
+                                dismissBottomSheet();
+                                Clipboard.setString(content);
+                            }}
+                            testID='at_mention.bottom_sheet.copy_code'
+                            text={intl.formatMessage({id: 'mobile.markdown.code.copy_code', defaultMessage: 'Copy Code'})}
+                        />
+                        <SlideUpPanelItem
+                            destructive={true}
+                            icon='cancel'
+                            onPress={() => {
+                                dismissBottomSheet();
+                            }}
+                            testID='at_mention.bottom_sheet.cancel'
+                            text={intl.formatMessage({id: 'mobile.post.cancel', defaultMessage: 'Cancel'})}
+                        />
+                    </View>
+                );
+            };
+
+            bottomSheet({
+                closeButtonId: 'close-code-block',
+                renderContent,
+                snapPoints: [bottomSheetSnapPoint(2, ITEM_HEIGHT, insets.bottom), 10],
+                title: intl.formatMessage({id: 'post.options.title', defaultMessage: 'Options'}),
+                theme,
+            });
+        }
+    }, [managedConfig, intl, insets, theme]);
```

Comment:
Not a big deal, but can be slightly improved if we pass `managedConfig?.copyAndPasteProtection !== 'true'` into the dependencies.